src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -428,6 +508,44 @@ partial class C
                 );
         }
 
+        [Fact]
+        public void PartialRecord_ParametersInScopeOfBothParts()
+        {
+            var src = @"
+var c = new C(2);
+System.Console.Write((c.P1, c.P2));
+
+public partial record C(int X)
+{
+    public int P1 { get; set; } = X;
+}
+public partial record C
+{
+    public int P2 { get; set; } = X;
+}
+";
+            var comp = CreateCompilation(new[] { src, IsExternalInitTypeDefinition }, options: TestOptions.DebugExe, parseOptions: TestOptions.RegularPreview);
+            comp.VerifyDiagnostics();
```

Comment:
>.VerifyDiagnostics() [](start = 16, length = 20)

Consider calling VarifyDiagnostics on result of CompileAndVerify. That set will include any warnings that might be reported by emit phase.